eZ/Publish/API/Repository/Values/URL/URL.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace eZ\Publish\API\Repository\Values\URL;
+
+use eZ\Publish\API\Repository\Values\ValueObject;
+
+class URL extends ValueObject
+{
+    /**
+     * @var int
+     */
+    protected $id;
+
+    /**
+     * @var string
```

Comment:
what format is the url expected in? _(same goes for structs, which you can probably remodel so all inherits from a base struct)_

Reply:
Done in c74b7acf267b60dea71b5b3798221957eb64ffbe